checkstyle.xml

```diff
@@ -30,7 +30,6 @@
     <module name="StaticVariableName"/>
     <module name="TypeName"/>
     <module name="AvoidStarImport"/>
-    <module name="IllegalImport"/>
```

Comment:
Why is this rule removed? I see why this is not needed, but I would like to know why, and this should also be removed in `checkstyle_text.xml`.

<revew, configuration